api/filters/suffix/suffix.go

```diff
@@ -48,3 +49,7 @@ func (f Filter) evaluateField(node *yaml.RNode) error {
 	return f.trackableSetter.SetScalar(fmt.Sprintf(
 		"%s%s", node.YNode().Value, f.Suffix))(node)
 }
+
+func DefaultFieldSpecs() types.FsSlice {
```

Comment:
builtinconfig.MakeDefaultConfig().NameSuffix seems to be handy enough to be called directly. Maybe not defining a new function for that.

Reply:
oh, I see!

Reply:
@natasha41575 Shall we surface the builtin const fieldspec var so users can import directly?

Reply:
The default fieldspecs are already surfaced here: https://github.com/kubernetes-sigs/kustomize/blob/8dab94964f7a3e4e9700ba9bd3eaba333c29be37/api/konfig/builtinpluginconsts/defaultconfig.go#L28

Rather than surfacing the default fieldspecs in each filter individually as done in this PR, I would suggest adding a new method to the above file, maybe something like `func GetDefaultFieldSpecFor(filter string) string`, or just use the existing method `GetDefaultFieldSpecsAsMap()` to get the default fieldspecs. IMO there is no need to create a new `DefaultFieldSpecs()` in each filter.

Reply:
+1 to  call `GetDefaultFieldSpecsAsMap()`